core/templates/dev/head/pages/exploration_editor/feedback_tab/FeedbackTab.js

```diff
@@ -176,11 +186,17 @@ oppia.controller('FeedbackTab', [
             var UNSAVED_CHANGES_MSG = 'You have unsaved changes to ' +
               'this exploration. Please save/discard your unsaved changes if ' +
               'you wish to accept.';
+            var ACTION_EDIT_SUGGESTION = 'edit';
+            $scope.stateName = stateName;
+            $scope.suggestion_id = suggestionId;
```

Comment:
use lowerCamelCasing in js.

Reply:
sorry, sir @DubeySandeep  but can you tell me for which variable I have to do lower camel case.
thanks